app/src/main/java/com/example/qrcodescanner/UIscreen.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.qrcodescanner
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.widget.Toast
+
+class UIscreen : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_u_iscreen)
+    }
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.nav_menu,menu)
+
+        return true
+    }
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        when(item.itemId){
+            R.id.nav_acceuil -> Toast.makeText(this,"acceuil", Toast.LENGTH_SHORT).show()
```

Comment:
please add the string to strings.xml ( see the documentation bellow ) for both acceuil and quitter